src/System.Collections.Specialized/src/System/Collections/Specialized/NameObjectCollectionBase.cs

```diff
@@ -61,6 +88,152 @@ protected NameObjectCollectionBase(int capacity)
             Reset(capacity);
         }
 
+#if netstandard17
```

Comment:
All these #ifdef's can be eliminated as well if we remove the other build configurations.